upcoming_events.yml

```diff
@@ -0,0 +1,27 @@
+- event: VM Workshop 2021
```

Comment:
@jackjia-ibm the file path need to be `_data/upcoming_events.yml`. Currently the file is getting stored in root directory.

I think we will need to make change to this step [here](https://github.com/zowe/zowe.github.io/blob/master/.github/workflows/update_upcoming_events.yml#L32):
```
# Runs the Script to convert ics to yaml format
      - name: Run ics-to-yml.py
      - run:  | 
            python master/_data/ics-to-yml.py
```

- `cd _data/`
- `python ics-to-yml.py` when run locally works fine

Reply:
@jackjia-ibm Currently when we are running `python _data/ics-to-yml.py`. It's creating `upcoming_events.yml` in the root directory.
I think if we can `cd _data` and then execute the script (`python ics-to-yml.py`). It might generate the desired file as `_data/upcoming_events.yml`

Reply:
https://github.com/zowe/zowe.github.io/pull/143 should fix this. https://github.com/zowe/zowe.github.io/pull/144 is example PR it created.